test/common.py

```diff
@@ -331,22 +336,39 @@ def run_train_translate(train_params: str,
         cp_metrics = cp_decoder.decode_and_evaluate()
         logger.info("Checkpoint decoder metrics: %s", cp_metrics)
 
+        # import shutil
+        # shutil.copytree(model_path, "/Users/post/code/sockeye/t/model")
+
         logger.info("Translating with parameters %s.", translate_params)
         # Translate corpus with the 1st params
         out_path = os.path.join(work_dir, "out.txt")
-        params = "{} {} {}".format(sockeye.translate.__file__,
-                                   _TRANSLATE_PARAMS_COMMON.format(model=model_path,
-                                                                   input=test_source_path,
-                                                                   output=out_path,
-                                                                   quiet=quiet_arg),
-                                   translate_params)
+        translate_score_path = os.path.join(work_dir, "out.scores.txt")
+        params = "{} {} {} --output-type translation_with_score".format(sockeye.translate.__file__,
+                                                                        _TRANSLATE_PARAMS_COMMON.format(model=model_path,
+                                                                                                        input=test_source_path,
+                                                                                                        output=out_path,
+                                                                                                        quiet=quiet_arg),
+                                                                        translate_params)
 
         if test_source_factor_paths is not None:
             params += _TRANSLATE_WITH_FACTORS_COMMON.format(input_factors=" ".join(test_source_factor_paths))
 
         with patch.object(sys, "argv", params.split()):
             sockeye.translate.main()
 
+        # Break out translation and score
+        outputs = open(out_path).readlines()
+        with open(out_path, 'w') as out_translate, open(translate_score_path, 'w') as out_scores:
+            for output in outputs:
+                output = output.strip()
+                try:
+                    score, translation = output.split('\t')
+                except:
```

Comment:
lets avoid bare except statements. This should be a ValueError, but why would we expect this to fail? The test is in control of the output type. Lets remove the try-except alltogether

Reply:
For blank lines (which are explicitly created so as to be tested), the output when using `translation_with_score` is just the score (so only a single field).